go.sum

```diff
@@ -262,6 +262,8 @@ github.com/russross/blackfriday v1.6.0 h1:KqfZb0pUVN2lYqZUYRddxF4OR8ZMURnJIG5Y3V
 github.com/russross/blackfriday v1.6.0/go.mod h1:ti0ldHuxg49ri4ksnFxlkCfN+hvslNlmVHqNRXXJNAY=
 github.com/scalr/go-scalr v0.0.0-20240902160450-c71eed0dd7f4 h1:sPrrzxsDcHODrum/dgM931ux41BL+Ckn35av34yiGUw=
 github.com/scalr/go-scalr v0.0.0-20240902160450-c71eed0dd7f4/go.mod h1:p34SHb25YRvbgft7SUjSDYESeoQhWzAlxGXId/BbaSE=
+github.com/scalr/go-scalr v0.0.0-20240929210950-4e55784195ad h1:G+GZ3pm22r748k05/FBnBPKBmjG8xf7UMq1cEOkPG5Y=
```

Comment:
@igor-melnyk-scalr hashes of the previous versions must be removed

Reply:
done